src/test/datascience/interactiveWindow.vscode.test.ts

```diff
@@ -44,202 +44,253 @@ import { areInterpreterPathsSame } from '../../platform/pythonEnvironments/info/
 import { IPythonApiProvider } from '../../platform/api/types';
 import { isEqual } from '../../platform/vscode-path/resources';
 import { PythonEnvironment } from '../../platform/pythonEnvironments/info';
-
-suite('Interactive window', async function () {
-    this.timeout(120_000);
-    let api: IExtensionTestApi;
-    const disposables: IDisposable[] = [];
-    let interactiveWindowProvider: InteractiveWindowProvider;
-    let venNoKernelPath: vscode.Uri;
-    let venvKernelPath: vscode.Uri;
-    let pythonApiProvider: IPythonApiProvider;
-    let originalActiveInterpreter: PythonEnvironment | undefined;
-
-    setup(async function () {
-        if (IS_REMOTE_NATIVE_TEST()) {
-            return this.skip();
-        }
-        traceInfo(`Start Test ${this.currentTest?.title}`);
-        api = await initialize();
-        interactiveWindowProvider = api.serviceManager.get(IInteractiveWindowProvider);
-        pythonApiProvider = api.serviceManager.get<IPythonApiProvider>(IPythonApiProvider);
-        traceInfo(`Start Test (completed) ${this.currentTest?.title}`);
-    });
-    teardown(async function () {
-        traceInfo(`Ended Test ${this.currentTest?.title}`);
-        if (this.currentTest?.isFailed()) {
-            // For a flaky interrupt test.
-            await captureScreenShot(`Interactive-Tests-${this.currentTest?.title}`);
+import { EXTENSION_ROOT_DIR } from '../../platform/constants.node';
+
+type DebuggerType = 'VSCodePythonDebugger' | 'JupyterProtocolDebugger';
+const debuggerTypes: DebuggerType[] = ['JupyterProtocolDebugger', 'VSCodePythonDebugger'];
+debuggerTypes.forEach((debuggerType) => {
```

Comment:
Running the IW debugger tests against both old and new jupyter protocol debugger.